src/pybind/mgr/dashboard/.pylintrc

```diff
@@ -16,7 +16,7 @@ ignore-patterns=
 
 # Python code to execute, usually for sys.path manipulation such as
 # pygtk.require().
-init-hook='import sys; sys.path.append("./")'
+init-hook='import sys; sys.path.append("./"); sys.path.append("../../../python-common")'
```

Comment:
Weird... why is this suddenly failing? We have pylint pinned 🤯

Reply:
BTW, I think we should ["prepend"](https://stackoverflow.com/questions/31414041/how-to-prepend-a-path-to-sys-path-in-python) this, in order to give it prevalence over any Python ceph package. I remember that in some Jenkins nodes, previous builds left an older version of the `python-common.ceph` package installed in `site-packages`...

Reply:
BTW, I see that we could do this in the `tox.ini` `setenv=PYTHONPATH=...`. At least that's for example what Adam/Cephadm folks did for their `tox.ini`.

Reply:
I tried tracking down what changed yesterday for hours. Possible culprits were `pip` or a few other unpinned libs but I could not find a smoking gun. FWIW
Note that running `python` in the `lint` testenv that runs pylint was able to import these modules just fine. The issue was specific to pylint.